R/rdataretriever.R

```diff
@@ -759,8 +758,19 @@ install_retriever <- function(method = "auto", conda = "auto") {
 # global reference to python modules (will be initialized in .onLoad)
 retriever <- NULL
 
-.onLoad <- function(libname, pkgname) {
-  ## assignment in parent environment!
-  retriever <<- reticulate::import("retriever", delay_load = TRUE)
-  retriever_meets_min_version()
+.onLoad = function(libname, pkgname) {
+    if(reticulate::py_available()){
+        # install_retriever()
+        install_python_modules <- function(method = "auto", conda = "auto") {
```

Comment:
We removed this part of code but I think we may need to test if python is available and retriever is available.